src/pages/version/ActionButtons.tsx

```diff
@@ -75,7 +77,7 @@ export const ActionButtons: React.FC<ActionButtonProps> = ({
         <ScheduleTasks
           versionId={versionId}
           isButton
-          disabled={isPatchOnCommitQueue}
+          disabled={isPatchOnCommitQueue && activated}
```

Comment:
should this be isPatchOnCommitQueue || activated?

Reply:
I don't think so. `isPatchOnCommitQueue` is true for unactivated mainline commits so `isPatchOnCommitQueue || activated` will disable the button when it shouldn't.